Plugin/ConfigFieldPlugin.php

```diff
@@ -88,23 +88,17 @@ public function afterGetTooltip(Subject $subject, $result)
      * @param string $result
      * @return string
      */
-    public function afterGetComment(Subject $subject, $result)
-    {
-        $resultIsPhrase = $result instanceof Phrase;
-        if ($resultIsPhrase) {
-            $phrase = $result;
-            $result = $phrase->getText();
-            $arguments = $phrase->getArguments();
+    public function afterGetComment(Subject $subject, $result) {
```

Comment:
Please move the `{` back to the next line.

Reply:
Fixed it